main.star

```diff
@@ -10,6 +10,11 @@ cdk_erigon_package = "./cdk_erigon.star"
 databases_package = "./databases.star"
 deploy_zkevm_contracts_package = "./deploy_zkevm_contracts.star"
 ethereum_package = "./ethereum.star"
+optimism_package = (
+    "github.com/leovct/optimism-package/main.star@feat/deploy-to-external-l1"
+)
+# TODO: Use the official repository once the feature to deploy the OP package to an external L1 is merged.
```

Comment:
Let's not merge this PR until OP package `v1.2.0` is released.

Reply:
https://github.com/ethpandaops/optimism-package/releases/tag/1.2.0 just got released! migration to the official op kurtosis package done in this commit: [de54cd1](https://github.com/0xPolygon/kurtosis-cdk/pull/418/commits/de54cd120041930b68b96b79d594255fafc605f6)